docs/3.0rc/resources/upgrade-prefect-3.mdx

```diff
@@ -202,42 +203,33 @@ def my_flow():
 
 ## Task caching and result persistence
 
-Prefect 3 supports transactional semantics&mdash;tasks runs can be grouped into larger transactions, enabling more resilient workflows.
-In keeping with the transactional model, Prefect 3 encourages task runs to be idempotent&mdash;intended to run only once per flow run.
+Prefect 3 supports transactional semantics &mdash; tasks runs can be grouped into larger transactions, enabling more resilient workflows.
+In keeping with the transactional model, Prefect 3 encourages task runs to be idempotent &mdash; intended to run only once per flow run.
 
 To support this, Prefect 3 has more eager, intelligent task caching defaults than Prefect 2.
 Whenever a task runs, Prefect 3 first computes its cache key, then uses that cache key to evaluate whether that same task has run with the same inputs and as part of the same flow before. 
-If it has, the cached result from the previous run is used. If it hasn't, the task runs. 
+If it has, the cached result from the previous run is used. If it hasn't, the task runs.
 
-This means that tasks that may have re-run with the same inputs in Prefect 2 do not re-run in Prefect 3 unless the cache key is modified to be more precise.
+Tasks that may have re-run with the same inputs in Prefect 2 do not re-run in Prefect 3 unless the cache key is modified to be more precise.
 The cache key can be configured through the new global `cache_policy` setting, or through the `cache_policy` on a particular task.
 If a particular task cannot be made idempotent, you can disable this behavior by setting `cache_policy=NONE` on the task.
 
-Results and cache keys are not stored with your Prefect Cloud account or Prefect server instance. 
-They are persisted entirely in your configured result location. 
-
 This caching behavior only takes effect when result persistence is enabled.
-The simplest way to enable result persistence is through the Prefect setting `PREFECT_RESULTS_PERSIST_BY_DEFAULT=true`.
-
-See the [caching documentation](/3.0rc/develop/write-tasks#caching) for more information.
-
-## Data validation with Pydantic V2
-
-Prefect 2 uses Pydantic V1 to validate data, particularly for blocks and work pools.
-
-Prefect 3 uses Pydantic V2. This upgrade does not require changes for any built-in Prefect interfaces or supported integrations. 
```

Comment:
Is there a reason to remove this, as it could require user changes? I think if people are passing in their own pydantic v1 models, they would run into issues.